tests/JiraClientStub.ts

```diff
@@ -7,7 +7,8 @@ export const jiraClientStub = {
     switch (issueId) {
       case 'MMF-1111': return { key: 'MMF-1111', fields: { project: { key: 'MMF' }, issuetype: { name: 'Epic' } } };
       case 'KEY-1111': return { key: 'KEY-1111', fields: { project: { key: 'KEY' }, issuetype: { name: 'Epic' } } };
-      case 'KEY-1234': return { key: 'KEY-1234', fields: { project: { key: 'KEY' }, issuetype: { name: 'Task' } } };
+      case 'KEY-1234': return { key: 'KEY-1234', fields: { project: { key: 'KEY' }, issuetype: { name: 'Task' }, creator: {displayName: "CreatorKEY1234"} } };
```

Comment:
It's "normal" user name, therefore with spaces
```suggestion
      case 'KEY-1234': return { key: 'KEY-1234', fields: { project: { key: 'KEY' }, issuetype: { name: 'Task' }, creator: {displayName: "Creator of KEY-1234"} } };
```